src/structures/idt.rs

```diff
@@ -593,17 +604,41 @@ impl<T> PartialEq for Entry<T> {
 }
 
 /// A handler function for an interrupt or an exception without error code.
```

Comment:
I think we should also mention here that the type requires the `x86-interrupt` feature. Maybe something like this:
```suggestion
/// A handler function for an interrupt or an exception without error code.
///
/// This type alias is only usable with the `abi_x86_interrupt` feature enabled.
```

(This also applies to the other type aliases.)